src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/AcrylicBrush.Rendering.Blur.Android.cs

```diff
@@ -0,0 +1,272 @@
+		//	{
+		//		return;
+		//	}
+
+		//	var formsView = MaterialFrame.AndroidBlurRootElement;
+		//	var renderer = Platform.GetRenderer(formsView);
+		//	if (renderer == null)
+		//	{
+		//		return;
+		//	}
+
+		//	bool IsAncestor(Element child, Layout parent)
+		//	{
+		//		if (child.Parent == null)
+		//		{
+		//			return false;
+		//		}
+
+		//		if (child.Parent == parent)
+		//		{
+		//			return true;
+		//		}
+
+		//		return IsAncestor(child.Parent, parent);
+		//	}
+
+		//	if (!IsAncestor(MaterialFrame, MaterialFrame.AndroidBlurRootElement))
+		//	{
+		//		throw new InvalidOperationException(
+		//			"The AndroidBlurRootElement of the MaterialFrame should be an ancestor of the MaterialFrame.");
+		//	}
+
+		//	Platform.SetRenderer(formsView, renderer);
+		//	_blurRootView = renderer.View;
+
+		//	_realtimeBlurView?.SetRootView(_blurRootView);
+		//}
+
+		private void UpdateAndroidBlurOverlayColor(bool invalidate = true)
+		{
+			if (IsAndroidBlurPropertySet)
+			{
+				var tintColorWithOpacity =
+					Color.FromArgb(
+						(byte)(TintOpacity * TintColor.A),
+						TintColor.R,
+						TintColor.G,
+						TintColor.B);
+				Android.Graphics.Color tintColor = tintColorWithOpacity;
+				_realtimeBlurView?.SetOverlayColor(tintColor, invalidate);
+			}
+		}
+
+		private void UpdateAndroidBlurRadius(bool invalidate = true)
+		{
+			if (IsAndroidBlurPropertySet)
+			{
+				//TODO
+				//_realtimeBlurView?.SetBlurRadius(
+				//	ContextHelper.Current.ToPixels(AndroidBlurRadius),
+				//	invalidate);
+			}
+		}
+
+		private void UpdateMaterialBlurStyle(bool invalidate = true)
+		{
+			if (_realtimeBlurView == null || IsAndroidBlurPropertySet)
+			{
+				return;
+			}
+
+			//TODO
+			//_realtimeBlurView.SetBlurRadius(
+			//	ContextHelper.Current.ToPixels(StyledBlurRadius),
+			//	invalidate);
+
+			Android.Graphics.Color color = DarkBlurOverlayColor;
+			_realtimeBlurView.SetOverlayColor(color, invalidate);
+		}
+
+		private void EnableBlur(ViewGroup view)
+		{
+			if (this.Log().IsEnabled(LogLevel.Information))
+			{
+				this.Log().LogInformation("Renderer::EnableBlur()");
+			}
+
+			if (_realtimeBlurView == null)
+			{
+				_realtimeBlurView = new RealtimeBlurView(view.Context);
+			}
+
+			UpdateAndroidBlurRadius();
+			UpdateAndroidBlurOverlayColor();
+			UpdateMaterialBlurStyle();
+			//UpdateAndroidBlurRootElement();
+
+			_realtimeBlurView.SetDownsampleFactor(CurrentBlurRadius <= 10 ? 1 : 2);
+
+			UpdateCornerRadius();
+
+			if (view.ChildCount > 0 && ReferenceEquals(view.GetChildAt(0), _realtimeBlurView))
+			{
+				// Already added
+				return;
+			}
+
+			if (this.Log().IsEnabled(LogLevel.Information))
+			{
+				this.Log().LogInformation("Renderer::EnableBlur() => adding pre draw listener");
+			}
+
+			//view.AddView(
+			//	_realtimeBlurView,
+			//	0,
+			//	new FrameLayout.LayoutParams(
+			//		ViewGroup.LayoutParams.MatchParent,
+			//		ViewGroup.LayoutParams.MatchParent,
+			//		GravityFlags.NoGravity));
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)